src/__tests__/MissionsProfileComponent.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { useSelector } from 'react-redux';
+import '@testing-library/jest-dom';
+import MissionsProfileComponent from '../components/MissionsProfileComponent';
+
+// Mock useSelector
+jest.mock('react-redux', () => ({
+  ...jest.requireActual('react-redux'),
+  useSelector: jest.fn(),
+}));
+
+describe('MissionsProfileComponent', () => {
+  const mockMissionsArray = [
+    {
+      mission_id: '1',
+      mission_name: 'Mission 1',
+      reserved: true,
+    },
+    {
+      mission_id: '2',
+      mission_name: 'Mission 2',
+      reserved: false,
+    },
+    {
+      mission_id: '3',
+      mission_name: 'Mission 3',
+      reserved: true,
+    },
+  ];
+
+  beforeEach(() => {
+    // eslint-disable-next-line max-len
```

Comment:
- [ ] Please don't disable the `eslinter` in your project, if you are getting an eslinter issue then you need to resolve the issue and try to understand, why the error is coming and what you can do to optimize your code for no error.